doc/docs/reference/patient.md

```diff
@@ -11,3 +11,9 @@ Returns a new `Episode` for this patient.
 Return the dictionary representation of this patient - suitable for serialization.
 
     patient.to_dict(user)
+
+
+#### bulk_update
+pass in a dictionary of subrecords you want to update, pass in an episode if one exists.
```

Comment:
So... I would struggle to call this method based on this! A code snippet would be great, and mentioning that User is a required argument...